### DISCUSSION TITLE
Miter inflation

### DISCUSSION THREAD

**Discussion by dm356** [google/s2geometry#384] [Q&A] [answered]
I see that the S2BufferOperation class allows us to buffer polygons with a circle sweep.  Is there an easy way to do a miter inflation (like in here: https://stackoverflow.com/questions/1109536/an-algorithm-for-inflating-deflating-offsetting-buffering-polygons)?

**Comment by smcallis** [comment, marked as answer]
We don't support it because a circular sweep it equivalent to finding all the points within a certain distance of the input geometry, which is effectively always what you want for GIS applications.  There's no reason it couldn't be done but you'd have to offset the edges and generate the intersection points manually, passing them to S2Builder.

**Reply by dm356** [reply]
Ok.  Understood.  Thanks.
